charts/apbs-rest/values.yaml

```diff
@@ -8,12 +8,8 @@ common:
   env:
     googleAnalyticsId:
     googleAnalyticsJobidIndex:
-
-tesk:
-  image:
-    repository: eu.gcr.io/tes-wes/tesk-api
-    tag: v0.4.2
-    pullPolicy: IfNotPresent
+    minioAccessKey: admin
+    minioSecretKey: adminadmin
```

Comment:
Are we storing the username and password in these publicly available files on github?
If so, that should change and not be deployed.

Reply:
Good catch... I hope those are placeholders we replace with information stored locally!

Reply:
That's right. The values shown are placeholders, replaced by whatever one defines either on the install command or within a passed-in values file

Reply:
That's great. Problem resolved!